content/en/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md

```diff
@@ -91,27 +87,27 @@ Although, Poseidon-Firmament scheduler is capable of scheduling various types of
 |Support for Max-Pod per node|Y|Y|Poseidon-Firmament scheduler seamlessly co-exists with Kubernetes default scheduler.|
 |Support for Ephemeral Storage, in addition to CPU/Memory|Y|Y||
 
+## Installation
 
-## Installation	
-
-For in-cluster installation of Poseidon, please start at the [Installation instructions](https://github.com/kubernetes-sigs/poseidon/blob/master/docs/install/README.md).
+[Installation](https://github.com/kubernetes-sigs/poseidon/blob/master/docs/install/README.md#Installation) explains how to deploy Poseidon-Firmament to your cluster.
```

Comment:
nit: Line 92, This change makes this sentence is vague, `Installation explains how to `. What installation?
The [Poseidon-Firmament installation guide](...) explains how to deploy ...